src/main/java/minijava/parser/Parser.java

```diff
@@ -467,6 +468,8 @@ private void parseNewObjectArrayExpression() {
           case LBRACKET:
             parseNewArrayExpression();
             break;
+          default:
+            unexpectCurrentToken();
```

Comment:
Ah, we're using this quite often. Maybe we should address this separately, also quality of error messages isn't paramount.